docs/source/scenarios/citi_bike.rst

```diff
@@ -753,3 +753,65 @@ random mode, we hope this could help you learn how to use the environment interf
 
 Jump to `this notebook <https://github.com/microsoft/maro/tree/master/notebooks/bike_repositioning/interact_with_environment.ipynb>`_
 for a quick experience.
+
+~~~~~~
+
+Inter-view
+~~~~~~~~~~
+
+For Inter-view, user could view top-k stations with different
+attributes.
+
+.. figure:: ..\images\visualization\dashboard\citi_bike_inter_epoch.gif
+   :alt: citi\_bike\_inter\_epoch
+
+Intra-view
+~~~~~~~~~~
+
+Intra-view is divided into two dimensions according to time and space.
+
+If user choose to view information by station, it means that attributes
+of all snapshots within a selected station would be displayed. By
+changing the option "station index", user could view data of different
+stations. By changing the option "Snapshot Sampling Ratio", Users can
+freely adjust the sampling rate. For example, if there are 100 snapshots
+and user selected 0.3 as sampling ratio, 30 snapshots data would be
+selected to render the chart.
+
+.. figure:: ..\images\visualization\dashboard\citi_bike_inter_epoch_by_station.gif
+   :alt: citi\_bike\_inter\_epoch\_by\_station
+
+To be specific, the line chart could be customized with operations in
+the following example.
+
+By choosing the item "All", all of attributes would be displayed. In
+addition, according to the data characteristics of each scenario, users
+will be provided with the option to quickly select a set of data.
+
+e.g. In this scenario, item "Requirement Info" refers to
+[trip\_requirement, shortage, fulfillment].
+
+.. figure:: ..\images\visualization\dashboard\citi_bike_inter_epoch_by_station_2.gif
+   :alt: citi\_bike\_inter\_epoch\_by\_station\_2
+
+Moreover, to improve the flexibility of visualizing data, user could use
+pre-defined formula and selected attributes to generate new attributes.
+Generated attributes would be treated in the same way as origin
+attributes.
+
+.. figure:: ..\images\visualization\dashboard\citi_bike_inter_epoch_by_station_3.gif
+   :alt: citi\_bike\_inter\_epoch\_by\_station\_3
+
+If user choose to view information by snapshot, it means attributes of
+all holders within a selected snapshot would be displayed. By changing
+option "snapshot index", user could view data of different snapshot. By
+changing option "Snapshot Sampling Ratio", user could change the number
+of sampled data.
+
+Particularly, if user want to check the name of a specific
+holder(station in this scenario), just hovering on the according bar.
```

Comment:
Maybe you can mention that ``the resource holder in this scenario is the bike station`` in the beginning of this part, and then use ``stations`` instead of ``holders`` later?